core/Piranha.Manager/Services/ContentTypeService.cs

```diff
@@ -267,6 +267,7 @@ private async Task<RegionItemModel> CreateRegionAsync(ContentTypeBase type, stri
             {
                 field.Model = (IField)regionModel;
                 field.Meta.NotifyChange = true;
+                field.Meta.Settings = fieldType.Settings;
```

Comment:
Isn't this already handled at line 255? My guess is that you solved the underlying problem in the `ContentTypeBuilder`.

Reply:
Oops, yes you're right! I was tinkering on an older version of Piranha that didn't have the assignment to settings and must have overlooked this. 

I have removed this line in the latest commit.